CustomDropDown/View/CustomDropDown.swift

```diff
@@ -12,6 +12,18 @@ class CustomDropDown<T>: UIView {
     
     // MARK: - Variables
     
+    public var cornerRadius: CGFloat = 0.0 {
```

Comment:
As we are giving option for corner radius in the config, do we need this property public?

Any specific reason? Please explain.

Reply:
It was my bad, sorry. Fixed in last commit!